test/functional/rpc_getblockstats.py

```diff
@@ -160,6 +160,9 @@ def run_test(self):
         assert_raises_rpc_error(-1, 'getblockstats hash_or_height ( stats )', self.nodes[0].getblockstats, '00', 1, 2)
         assert_raises_rpc_error(-1, 'getblockstats hash_or_height ( stats )', self.nodes[0].getblockstats)
 
+        self.log.info('Test block height 0')
+        genesis_stats = self.nodes[0].getblockstats(0)
+        assert_equal(genesis_stats["blockhash"], "0f9188f13cb7b2c71f2a335e3a4fc328bf5beb436012afca590b1a11466e2206")
```

Comment:
Would be nice to test the other corner-case result values

Reply:
I am not sure if you had more values in mind (let me know) but I have added the old and new values that this PR touches.

Reply:
Added a test with an `OP_RETURN` output.